pulp_container/app/tasks/sync_stages.py

```diff
@@ -355,9 +356,9 @@ async def handle_blobs(self, manifest_dc, content_data):
             await self.put(blob_dc)
         layer = content_data.get("config", None)
         if layer:
-            blob_dc = self.create_blob(layer, deferred_download=False)
-            manifest_dc.extra_data["config_blob_dc"] = blob_dc
-            await self.put(blob_dc)
+            config_blob_dc = await self.create_config_blob(layer)
+            manifest_dc.extra_data["config_blob_dc"] = config_blob_dc
+            await self.put(config_blob_dc)
```

Comment:
If you do not mind, please, rename `layer` to something self-explanatory, like `config`.